packages/shylu/shylu_dd/frosch/src/Adapters/Thyra_FROSchFactory_def.hpp

```diff
@@ -422,7 +422,7 @@ namespace Thyra {
                     repeatedMap = rcp_dynamic_cast<ConstXMap>(xTpetraRepeatedMap);
                 } else {
 #ifdef HAVE_SHYLU_DDFROSCH_EPETRA
-                    if (comm->getRank()==0) std::cout << "FROSch::FROSchFactory : WARNING: Cannot retrieve Epetra objects from ParameterList. Use Xpetra isntead." << std::endl;
+                    if (comm->getRank()==0) std::cout << "FROSch::FROSchFactory : WARNING: Cannot retrieve Epetra objects from ParameterList. Use Xpetra instead." << std::endl;
```

Comment:
1. Warnings should go to `std::cerr`.
2. Is this actually an error?  Should processing stop at this point?

Reply:
1. Thank you. I changed this.
2. There is an ASSERT afterwards which will stop it.